frontend/packages/network-attachment-definition-plugin/src/types/types.ts

```diff
@@ -15,12 +13,3 @@ export type NetworkAttachmentDefinitionConfig = {
   ipam?: IPAMConfig;
   plugins?: NetworkAttachmentDefinitionConfig[];
 };
-
-// The config is a JSON object with the NetworkAttachmentDefinitionConfig type stored as a string
-export type NetworkAttachmentDefinitionSpec = {
-  config: string;
-};
-
-export type NetworkAttachmentDefinitionKind = {
```

Comment:
why remove the type ? IMO its good to have it and use where appropriate

Reply:
I misunderstood your previous comment. I changed it back.